openfisca_us/variables/gov/states/mo/tax/income/mo_income_tax.py

```diff
@@ -0,0 +1,27 @@
+from openfisca_us.model_api import *
+
+
+class mo_income_tax(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO income tax"
+    unit = USD
+    documentation = "Missouri State income tax."
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-1040%20Print%20Only_2021.pdf"
+
+    def formula(tax_unit, period, parameters):
+        in_mo = tax_unit.household("state_code_str", period) == "MO"
+
+        income_tax_before_credits = tax_unit(
+            "mo_income_tax_before_credits", period
+        )
+
+        CREDITS = [
```

Comment:
Could be neater to make this a parameter (see the IRS refundable credit list for an example)

Reply:
blame @MaxGhenis for doing it this way for MA, but noted!